apps/cloudfront-functions/src/viewer-request-handler.ts

```diff
@@ -15,9 +15,15 @@ const handler = (
 
     // Check if the uri refers to a gitbook assets
     const isGitbookAssets = uri.startsWith('/gitbook/docs');
+    const isWoff2 = uri.endsWith('.woff2'); // woff2 is a font format
 
     // Add the .html extension if missing
-    if (!uri.endsWith('/') && !isGitbookAssets && !/\.[a-zA-Z]+$/.test(uri)) {
+    if (
+      !uri.endsWith('/') &&
+      !isGitbookAssets &&
+      !isWoff2 &&
+      !/\.[a-zA-Z]+$/.test(uri)
```

Comment:
I would generalize it as follows:

```suggestion
      !/\.[a-zA-Z0-9]+$/.test(uri)
```